src/insns/gcmode_32bit.adoc

```diff
@@ -0,0 +1,30 @@
+<<<
+
+[#GCMODE,reftext="GCMODE"]
+==== GCMODE
+
+Synopsis::
+Capability get CHERI execution mode
+
+Mnemonic::
+`gcmode rd, cs1`
+
+Encoding::
+include::wavedrom/gcmode.adoc[]
+
+Description::
+Decode the architectural capability CHERI execution mode from `cs1` and write
+the result to `rd`. It is not required that the input capability `cs1`  has its
+tag set to 1.
+
```

Comment:
it's worth mentioning that 0-cap mode, 1-int mode - i.e. clearly state what it returns

Reply:
Fixed